### DISCUSSION TITLE
【solved】Question about active the yml file

### DISCUSSION THREAD

**Discussion by shengjiayuu** [UBC-MDS/airline-customer-satisfaction-predictor#25] [General]
Does anyone know how to solve this issue? I have the environment active, but I am unable to connect it to a kernel.
<img width="293" alt="Screen Shot 2024-11-28 at 4 10 34 PM" src="https://github.com/user-attachments/assets/f830b7fb-6d2b-4f9d-9c37-0ce62e3424e0">

<img width="498" alt="Screen Shot 2024-11-28 at 4 09 09 PM" src="https://github.com/user-attachments/assets/384537b4-0af2-4045-bd91-aeb53d071053">

<img width="199" alt="Screen Shot 2024-11-28 at 4 05 02 PM" src="https://github.com/user-attachments/assets/173ba975-afa9-47ff-97cd-4581f1a636e0">

**Comment by cs-uche** [comment]
Hey @shengjiayuu are you still encountering this error? Can you please share more about the error

**Reply by shengjiayuu** [reply]
Heyyy @cs-uche nooo, it's all good now. I forgot how I solve it lol.
